pythonx/UltiSnips/snippet_manager.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, expand_trigger, forward_trigger, backward_trigger):
         enable_snipmate = True
         if _vim.eval("exists('g:UltiSnipsEnableSnipMate')") == "1":
             enable_snipmate = _vim.eval("g:UltiSnipsEnableSnipMate")
-        if enable_snipmate:
+        if enable_snipmate == "1":
```

Comment:
frankly speaking I don't know, what is idiomatic way to this in vim (handling bool values in config), but from my simple experiments vim.eval always returns string no matter is it number or not - so expecting exactly == "1" in python will accept let x = 1 and x = "1"
and everything else will be treated as false 

from VIM docs about vim.eval 
returns ... "a string if the Vim expression evaluates to a string or number"